mongo_items.py

```diff
@@ -0,0 +1,142 @@
+import requests, json, os
+from item import Item
+import sys
+import pymongo
+from datetime import datetime
+from bson.objectid import ObjectId
+
+def connect_to_mongo():
+    mongo_conn = os.environ["MONGO_CONN"]
+    client = pymongo.MongoClient(mongo_conn)
+    database = client[os.environ["MONGO_DB_NAME"]]
+
+    return database
+
+def get_items():
+    boardid = os.environ["MONGO_DB_NAME"]
+    collections = getListsOnBoards(boardid) # get the lists on a board
+    #setListIdInEnv(collections)
+
+    items = [] 
+    db = connect_to_mongo()
+
+    for collection in collections:
+        cards = db[collection].find()
+
+        for card in cards:
+             item_from_list = Item.from_raw_card(card)
+             items.append(item_from_list)
+
+    return items
+
+def getListsOnBoards(boardid):  
+    # Fetch collections from database
+    database = connect_to_mongo()
+    collections = database.list_collection_names()
```

Comment:
This doesn't work with your test.

Your app loops through the result of `database.list_collection_names()` when getting cards. But the test doesn't create the expected collections - `MONGO_LIST_TODO` etc. In fact, it creates just one collection, "test-collection" with an entry it in that would break your app if it tried to parse it as a todo item. And this solution also wouldn't work if you wanted to store other data types in your db.

Since you need those list names as environment variables anyway, why not just return them here from env vars rather than database.list_collection_names? And don't bother creating "test-collection"